docs/source/modules/api.rst

```diff
@@ -264,3 +264,20 @@ Classification Metrics
 
    train
    predict
+
+:mod:`dask_ml.datasets`: Datasets
+======================================================
+
+dask-ml provides some utilities for generating toy datasets.
+
+.. automodule:: dask_ml.datasets
+
+.. currentmodule:: dask_ml.datasets
+
+.. autosummary::
+   :toctree: generated/
+
+   make_counts
+   make_blobs
+   make_regression
+   make_classification_df
```

Comment:
Can you add `make_classification` here? It doesn't have a docstring yet (would be a good addition) but we'll want to make sure that it's included when one is added.

Reply:
Sure!